python/lsst/afw/math/functionLibrary.cc

```diff
@@ -61,8 +61,8 @@ void declarePolynomialFunctions(py::module &mod, const std::string & suffix) {
                BasePolynomialFunction2<ReturnT>>
                    clsPolynomialFunction2(mod, ("PolynomialFunction2" + suffix).c_str());
     /* PolynomialFunction2 Constructors */
-    clsPolynomialFunction2.def(py::init<unsigned int>());
     clsPolynomialFunction2.def(py::init<std::vector<double> const &>());
+    clsPolynomialFunction2.def(py::init<unsigned int>());
```

Comment:
Ah, this is what was causing one of the tests to fail. pybind11 was updated to allow scalar numpy arrays to be cast as integers or floats. Unlike a C++ compiler, which looks for the best overloaded function to match the parameters, pybind11 move linearly through the overloaded wrappers and uses the first function that it can cast the parameters into. So for overloaded functions, in pybind11 order matters.